cypress/integration/App.spec.js

```diff
@@ -0,0 +1,33 @@
+describe('App', () => {
```

Comment:
I really like the power Cypress brings to testing your testing suite. Thanks for running a quick demo for me!